.github/workflows/start-script.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# TODO: replace with master branch after merge
+curl -O- https://github.com/duckduckgo/sync-config/blob/lkruger/sync1/test/setup.sh
+curl -O- https://github.com/duckduckgo/sync-config/blob/lkruger/sync1/test/sync_predefined_autofill.json
+curl -O- https://github.com/duckduckgo/sync-config/blob/lkruger/sync1/test/sync_predefined_bookmarks.json
+curl -O- https://github.com/duckduckgo/sync-config/blob/lkruger/sync1/test/teardown.sh
+
+token="$(./setup.sh)"
```

Comment:
That script is being built at https://github.com/duckduckgo/sync-config/pull/1, and depends on a `test_user_setup` binary, being built at https://github.com/duckduckgo/sync_crypto/pull/8.

How is that binary going to get here?